src/js/edu-benefits/1990e/config/form.js

```diff
@@ -31,7 +48,36 @@ const formConfig = {
       pages: {
         applicantInformation: {
           path: 'applicant-information',
-          title: 'Applicant information'
+          title: 'Applicant information',
+          initialData: {},
+          uiSchema: {
+            applicantFullName: fullName.uiSchema,
+            applicantSocialSecurityNumber: ssn.uiSchema,
+            applicantDateOfBirth: currentOrPastDate.uiSchema('Applicant date of birth'),
```

Comment:
We've also in the past validated that applicants are at least 17. Should we do that here @goldenmeanie?